pages/linux/smem.md

```diff
@@ -0,0 +1,20 @@
+# smem
+
+> Show memory usage for programs.
+> More information: <https://manned.org/smem>.
+
+- Print memory usage for current processes:
+
+`smem`
+
+- Print memory usage for current processes for a every user on a system:
+
+`smem --users`
+
+- Print memory usage for current processes for a specified user:
+
+`smem --userfilter {{username}}`
```

Comment:
```suggestion
`smem --userfilter {{username}}`
```

It is not clear what regex type is supported from this code sample. What about to note about it in the description smth like (`{{regex-type}}` is used)?

Reply:
Sorry.

Reply:
No problem!